.eslintrc.js

```diff
@@ -0,0 +1,8 @@
+module.exports = {
+  "extends": ["airbnb-base", "prettier"],
```

Comment:
Out of curiosity -- why Airbnb?

Reply:
It's the most used lint configuration in Javascript projects, with very good documentation.